test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPublishTests2.cs

```diff
@@ -15,6 +17,31 @@ namespace Microsoft.Dnx.Tooling.FunctionalTests
     [Collection(nameof(ToolingFunctionalTestCollection))]
     public class DnuPublishTests2 : DnxSdkFunctionalTestBase
     {
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)] // This needs msbuild to run
+        [MemberData(nameof(DnxSdks))]
+        public void PublishWrappedProjectForSpecificFramework(DnxSdk sdk)
+        {
+            var solutionName = Path.Combine("DnuWrapTestSolutions", "WrapAndPublish");
+            var solution = TestUtils.GetSolution<DnuPublishTests2>(sdk, solutionName);
+
+            // Restore the wrapper project
+            sdk.Dnu.Restore(solution.GetWrapperProjectPath("ClassLibrary")).EnsureSuccess();
+
+            // Publish the console app
+            sdk.Dnu.Publish(solution.GetProject("DnxConsoleApp").ProjectFilePath, solution.ArtifactsPath, "--framework dnx451").EnsureSuccess();
+
+            // Get an SDK we can use the RUN the wrapped project (it has to be CLR because the project only supports dnx451)
+            var clrSdk = GetRuntime("clr", "win", "x86");
```

Comment:
Since we skip on Mono this could be safe. How about we create a quick shortcut on `DnuSdkFunctionalTestBase` just for this cause. I don't really think we should waste the effort of hiding this tuple.